src/ray/raylet/node_manager.cc

```diff
@@ -683,10 +694,10 @@ void NodeManager::GetObjectManagerProfileInfo() {
 void NodeManager::NodeAdded(const GcsNodeInfo &node_info) {
   const NodeID node_id = NodeID::FromBinary(node_info.node_id());
 
-  RAY_LOG(DEBUG) << "[NodeAdded] Received callback from node id " << node_id;
+  RAY_LOG(INFO) << "[NodeAdded] Received callback from node id " << node_id;
```

Comment:
` Received callback from node id` -> it is also very low level description. Can you explain in a high level description?